specification/ai/Azure.AI.Projects/evaluation-results/routes.tsp

```diff
@@ -0,0 +1,45 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@azure-tools/typespec-autorest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+import "../common/models.tsp";
+import "./models.tsp";
+import "../servicepatterns.tsp";
+
+using TypeSpec.Http;
+
+namespace Azure.AI.Projects;
+
+alias ListEvaluationResultsParameters = {
```

Comment:
As we discussed before, we need to filter by agent run Id, maybe we can add it as the parameter for list eval results API? To make it flexible, we can provide filter by "agent Id", "agent thread Id" and "agent run Id", then it'll help the UI team